.cci.jenkinsfile

```diff
@@ -20,13 +20,9 @@ pod(image: 'registry.fedoraproject.org/fedora:32', runAsUser: 0, kvm: true, memo
         """)
     }
 
-    stage("Build FCOS") {
-        shwrap("chown builder: /srv")
-        // just split into separate invocations to make it easier to see where it fails
-        cosa_cmd("init https://github.com/coreos/fedora-coreos-config")
-        cosa_cmd("fetch --strict")
-        cosa_cmd("build --strict")
-    }
+    shwrap("chown builder: /srv")
+    // Run stage Build FCOS (init, fetch and build)
+    fcosBuild(skipKola: 1, cosaDir: "/srv", noForce: 1)
```

Comment:
This is running as `root` whereas before it ran as `builder`. So then `fcosKola` fails with `EPERM`. I think we need to enhance `fcosBuild` and `fcosKola` to take a `user` parameter?

Ideally, we could just wrap all the bits we want to run as `builder` inside a `asUser("builder") { ... }`-type helper, though not sure how to implement that nicely.

Reply:
Even more ideally though, we'd split building the container and using it into two distinct phases. I think there's support with the OpenShift Jenkins plugins for building and pushing images to an imagestream, though I haven't dug into that much.